__tests__/index.test.js

```diff
@@ -0,0 +1,26 @@
+const index = require('../GitForWindowsHelper/index')
+
+process.env['GITHUB_WEBHOOK_SECRET'] = 'for-testing'
+
+test('reject requests other than webhook payloads', async () => {
+    const context = {
+        log: jest.fn(),
+        req: {
+            method: 'GET'
+        }
+    }
+
+    const expectInvalidWebhook = async (message) => {
+        context.log.mockClear()
+        expect(await index(context, context.req)).toBeUndefined()
+        expect(context.log).toHaveBeenCalledTimes(1)
+        expect(context.log.mock.calls[0][0].message).toEqual(message)
```

Comment:
`context.log()` gets called with a single string as an argument, doesn't it? Shouldn't this be `expect(context.log.mock.calls[0][0]).toEqual(message)` then?

Reply:
It is a single argument, but that argument is not a string, it is an `Error`. I've added a comment to explain that.